definitions/npm/@storybook/addon-knobs_v3.x.x/flow_all/addon-knobs_v3.x.x.js

```diff
@@ -0,0 +1,16 @@
+declare module "@storybook/addon-knobs/react" {
+  declare type Renderable = React$Element<any>;
+  declare type GroupId = string;
+
+  declare function array<T>(string, (Array<mixed> | {}), ?string, ?GroupId): Array<string>;
+  declare function boolean(string, boolean, ?GroupId): boolean;
+  declare function button(string, ((?{}) => void), ?GroupId): void;
```

Comment:
Is this syntax valid? ![](https://cloudup.com/cWEqer7p7_o+)

Reply:
What parser is giving you these errors?

Reply:
flow v0.73